eth-contracts/contracts/ServiceProviderFactory.sol

```diff
@@ -636,7 +642,7 @@ contract ServiceProviderFactory is RegistryContract {
     {
         require(
             _amount >= spDetails[_sp].minAccountStake,
-            "Minimum stake threshold exceeded");
+            "Minimum stake requirement not met");
```

Comment:
@roneilr is this the comment you were talking about?

Reply:
yes! this is correct - just was confusing to say that the "minimum threshold is exceeded" when the test is checking the opposite